tensorflow/contrib/lookup/lookup_ops.py

```diff
@@ -347,21 +348,21 @@ def __init__(self,
           use_node_name_sharing=use_node_name_sharing,
           key_dtype=key_dtype,
           value_dtype=value_dtype,
-          name=name)
+          name=self._name)
     else:
       self._table_ref = gen_lookup_ops.mutable_hash_table_of_tensors_v2(
           shared_name=shared_name,
           use_node_name_sharing=use_node_name_sharing,
           key_dtype=key_dtype,
           value_dtype=value_dtype,
           value_shape=self._default_value.get_shape(),
-          name=name)
+          name=self._name)
     super(MutableHashTable, self).__init__(key_dtype, value_dtype,
                                            self._table_ref.op.name.split(
                                                "/")[-1])
 
     if checkpoint:
-      saveable = MutableHashTable._Saveable(self, name)
+      saveable = MutableHashTable._Saveable(self, self._name)
```

Comment:
I believe this means that previous checkpoints cannot be read any more.

Reply:
If we use the same graph to load the checkpoint, `self._name` should be the same.
Will this break the functionality of checkpoint?